### DISCUSSION TITLE
could there please be a  up2date basic example?

### DISCUSSION THREAD

**Discussion by Jarrodsz** [sergiodxa/remix-auth#201] [General]
Could there please be a basic example?
I rigged the web for example they are outdated or authors add there own custom additions.
with several extra file includes which makes it not easy to understand.

Im trying to implement a basic remix-auth with form strategy in the most simple form.

It has been a real headache so far, an example would really help newbies to remix/react like me.

**Comment by sergiodxa** [comment]
Examples for Remix Auth are in the official [repository of Remix examples](https://github.com/remix-run/examples/tree/main/remix-auth-form).
